spec/provider/lambda_spec.rb

```diff
@@ -89,19 +99,21 @@
 
     context 'by creating a new function' do
       before do
-        provider.lambda.stub_responses(:list_functions, empty_list_functions_response)
+        provider.lambda.stub_responses(:get_function, 'ResourceNotFoundException')
         provider.lambda.stub_responses(:create_function, example_response)
       end
 
       example do
+
+        expect(provider).to receive(:log).with(/Function #{lambda_options[:function_name]} doesn't exists, creating\./)
         expect(provider).to receive(:log).with(/Created lambda: #{lambda_options[:function_name]}\./)
         provider.push_app
```

Comment:
Specs are failing because of the extra log info while trying to debug the horrible problems of options being parsed from yaml not working as intended.